src/app/home/components/home.component.html

```diff
@@ -1 +1,18 @@
-<button mat-raised-button (click)="openDialog()">Get Started!</button>
+<div class="top-spacer">
+  <p>Rock Paper Scissor</p>
+  <p>Let's solve your conflicts in the most elegantly</p>
+</div>
+
+<div class="top-spacer">
+  <p>Create a new game and invite your friends, huhu!</p>
+  <button mat-raised-button (click)="openDialog()">Get Started!</button>
+</div>
+
+<div class="top-spacer">
+  <p>Do you want to practice against the bot?</p>
+  <button mat-raised-button (click)="playAgainstTheBot()">Play Against The Bot!</button>
+</div>
+
+<div class="top-spacer">
+  <p>If you want to read intersting facts about the game, check this</p><a href="https://en.wikipedia.org/">link</a>
```

Comment:
Maybe keep the general over-the-top tone here with something like this:
`If you want to learn more about the ancient art of the game check out this link!`.

Also, probably it is better to link to the correct sub-page:

`https://en.wikipedia.org/wiki/Rock_paper_scissors`